src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -2,42 +2,60 @@
 	<system-page>
 		<div class="page-content">
 			<h1 class="featured-text">
-				Almost done...
+				One last thing!
 			</h1>
 			<p>
-				To complete your registration, please agree to the Terms of Use and Privacy Policy
+				{{ registrationMessage }}
 			</p>
 			<form
 				id="registerSocialTermsForm"
 				class="promptForm"
 				action="."
 				@submit.prevent.stop="postRegisterSocialForm"
 			>
-				<p class="terms">
-					<label>
-						<input
-							type="checkbox"
-							id="terms_agreement_popup"
-							name="terms_agreement_popup"
-							v-model="newAcctTerms"
-							@click="showNewAcctTermsError = validateTerms()"
-						>
+				<kv-base-input name="firstName"
+					class="fs-exclude"
+					type="text"
+					v-show="needsNames"
+					v-model.trim="firstName"
+					:validation="$v.firstName"
+				>
+					First name
+					<template #required>
+						Enter first name.
+					</template>
+				</kv-base-input>
+				<kv-base-input name="lastName"
+					class="fs-exclude"
+					type="text"
+					v-show="needsNames"
+					v-model.trim="lastName"
+					:validation="$v.lastName"
+				>
+					Last name
+					<template #required>
+						Enter last name.
+					</template>
+				</kv-base-input>
+				<kv-base-input name="newAcctTerms"
+					type="checkbox"
```

Comment:
I think it's cool that `type` can be set to checkbox like this. I think a future improvement for `kv-base-input` would be to take type as a prop and then display either a `kv-text-input` or `kv-checkbox`, maybe a `kv-radio` too since they have the Kiva Classic styling (and the default form styling looks very non-Kiva Classic in a Tailwind world).